e3sm_diags/plot/aerosol_aeronet_plot.py

```diff
@@ -108,4 +150,4 @@ def plot(
 
     plt.loglog(ref_site_arr, test_site_arr, "kx", markersize=3.0, mfc="none")
 
-    _save_plot(fig, parameter, PANEL_CFG, BORDER_PADDING)
+    _save_plot_aerosol_aeronet(fig, parameter)
```

Comment:
Use local _save_plot function to flexibly accommodate two plots with very different layouts.